rasa/core/featurizers/tracker_featurizers.py

```diff
@@ -524,15 +522,6 @@ def load(path: Text) -> Optional["TrackerFeaturizer"]:
 
     @staticmethod
     def _remove_action_unlikely_intent_from_states(states: List[State]) -> List[State]:
```

Comment:
I know it wasn't mentioned in the original PR review but I think we can remove the docstring for `_remove_action_unlikely_intent_from_events` as well.